<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List</class>
 <widget class="QWidget" name="List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>118</width>
    <height>324</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushClear">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Clear list</string>
       </property>
       <property name="icon">
        <iconset resource="tht.qrc">
         <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pushAdd">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Load tickers</string>
       </property>
       <property name="icon">
        <iconset resource="tht.qrc">
         <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushSave">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Save tickers</string>
       </property>
       <property name="icon">
        <iconset resource="tht.qrc">
         <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pushSaveAs">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Export tickers</string>
       </property>
       <property name="icon">
        <iconset resource="tht.qrc">
         <normaloff>:/images/export.png</normaloff>:/images/export.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QListWidget" name="list"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Tickers:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="labelElements">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>4</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="labelUnsaved">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>3</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>3</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushClear</tabstop>
  <tabstop>pushAdd</tabstop>
  <tabstop>pushSave</tabstop>
  <tabstop>pushSaveAs</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources>
  <include location="tht.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushClear</sender>
   <signal>clicked()</signal>
   <receiver>List</receiver>
   <slot>slotClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>List</receiver>
   <slot>slotSelectedItemChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushSave</sender>
   <signal>clicked()</signal>
   <receiver>List</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotClear()</slot>
  <slot>slotSelectedItemChanged()</slot>
  <slot>slotSave()</slot>
 </slots>
</ui>
